docs/spec/relayer/Chain.tla

```diff
@@ -78,12 +81,34 @@ ChannelUpdate(chainID, store, datagrams) ==
 
     chanCloseConfirmStore
 
+(***************************************************************************
+ Packet update operators
+ ***************************************************************************)
+\* Update the chain store of the chain with chainID and the packet log, 
+\* using the packet datagrams generated by the relayer
+\* (Handler operators defined in PacketHandlers.tla)    
+PacketUpdate(chainID, store, packetDatagrams, log) ==
+    \* if the sequence of packet datagrams is not empty
+    IF packetDatagrams /= AsSeqPacketDatagrams(<<>>)
```

Comment:
Why the structure is not the same as with others, i.e., it contains `HandlePacketRecv` and `HandlePacketAck` that internally check for the datagram type?